### PR TITLE
Using subprocess instead of os.system

### LINKED ISSUE #239: Replace os.system call in Python code with subprocess
I see that you are using a lot of `os.system()` in Python code. Python recommended `subprocess` functions over `os.system`.

### PR DESCRIPTION
**Pull request by prateekiiest**
Continuation of #245 
Fixes #239

### CODE REVIEW

File: access_point/server/server.py [hunk 1]
```diff
@@ -13,19 +13,19 @@ def install():
 
 @app.route('/config/<stt>/<tts>/<hotword>/<wake>')
 def config(stt, tts, hotword, wake):
-    os.system('sudo bash /home/pi/SUSI.AI/susi_linux/access_point/server/config.sh {} {} {} {}'.format(stt,tts,hotword,wake))  #nosec #pylint-disable type: ignore
+    subprocess.call(['sudo', 'bash' , '/home/pi/SUSI.AI/susi_linux/access_point/server/config.sh {} {} {} {}'.format(stt,tts,hotword,wake)])  #nosec #pylint-disable type: ignore
```
Comment (hongquan) [inline access_point/server/server.py]: Replace with

```py
subprocess.call(['sudo', 'bash' , '/home/pi/SUSI.AI/susi_linux/access_point/server/config.sh', stt, tts, hotword, wake])
```

Or

```py
subprocess.call(['sudo', 'bash' , 'server/config.sh', stt, tts, hotword, wake])
```

to not stick to location of _susi_linux_.

Comment (prateekiiest) [reply access_point/server/server.py]: ok @hongquan making the changes wherever this exists

File: main/speech/TTS.py [hunk 2]
```diff
@@ -29,9 +29,9 @@ def speak_flite_tts(text):
     # Call flite tts to reply the response by Susi
     flite_speech_file = config['flite_speech_file_path']
     print('flite -voice file://{0} -f {1}'.format(flite_speech_file, filename))
-    os.system(
-        'flite -v -voice file://{0} -f {1} -o extras/output.wav'.format(flite_speech_file, filename))
-    os.system('play extras/output.wav')
+    subprocess.call(
+        ['flite', '-v', '-voice', 'file://{0} -f {1} -o extras/output.wav'.format(flite_speech_file, filename)])
```
Comment (hongquan) [inline main/speech/TTS.py]: You still haven't split all command parameters here.

**Comment by hongquan** [comment]
Why do you make so many small commits which solve the same error?

image

There are a lot of errors similar to 0f92d73, why don't you fix at once?

**Comment by hongquan** [comment]
Later on we will replace all "sudo bash" call under access_point server.
It looks weird that the server can not be scripted to do the job of Bash (!?).

**Pull request merged**